XSOAR Engineer Training Pack (#29853)
demisto/content: Packs/XSOAR_Engineer_Training/README.md

# XSOAR Engineer Training 

> This pack contains content utilized to train you on how to be an XSOAR Engineer as part of the XSOAR Engineer Training Series from Cortex Customer Success.

Reviewer: ```suggestion
This pack contains content used to train you on how to be a Cortex XSOAR Engineer as part of the XSOAR Engineer Training Series from Cortex Customer Success.
```

## XSOAR 6.x Engineer Training Series

> You can find the XSOAR Engineer Training series for XSOAR 6.x on the Palo Alto Networks Live Community here:

Reviewer: ```suggestion
You can find the XSOAR Engineer Training series for Cortex XSOAR 6.x on the Palo Alto Networks Live Community here:
```
